### ISSUE TITLE
Save pump data to file, allow for easier setup after re-compiling Loop.

### ISSUE THREAD

**Issue by rdalisky** [LoopKit/Loop#226]
If one were able to save all Loop settings to an xml file, and then import the xml after Loop has been re-compiled, it would save time and allow for a more seamless upgrade process.

**Comment by ps2** [comment]
Settings are currently remembered when recompiling loop.

**Comment by loudestnoise** [comment]
@rdalisky The trick, at least as I've experienced it, is you want to use the same `MAIN_APP_BUNDLE_IDENTIFIER` you used before. When you build/run on your iPhone it will just replace the older one and like @ps2 said your settings should persist.

**Comment by rdalisky** [comment]
So don't delete the old app, just use the same identifier and install... hmm should have tried that!

**Comment by loudestnoise** [comment]
Yep, that's how you'd do it. Feel free to ask questions like this in Gitter as well. 

https://gitter.im/LoopKit/Loop

**Comment by CrushingT1D** [comment]
Settings could be stored in mLab (or possibly NS) each time a user makes a change. Then, we could read the latest settings the next time you connect. That would help here with this question when a user does need or want to change bundle IDs, but it would also allow to have a full history of settings stored for use in analysis in the future.

**Comment by apabari** [comment]
Wondering, should this issue be closed off?  Since settings are retained if you use the same identifier.

@CrushingT1D  - I'm wondering if an enhancement request should be raised for storing settings in NS or mLab, if enough folks want that functionality.
